keepassxc-browser/content/ui.js

```diff
@@ -324,16 +324,11 @@ const DOMRectToArray = function(domRect) {
 };
 
 const initColorTheme = function(elem) {
-    const colorTheme = kpxc.settings['colorTheme'];
-
-    if (colorTheme === undefined) {
-        elem.removeAttribute('data-color-theme');
-    } else if (colorTheme === 'system') {
-        const theme = window.matchMedia('(prefers-color-scheme: dark)').matches ? 'dark' : 'light';
-        elem.setAttribute('data-color-theme', theme);
-    } else {
-        elem.setAttribute('data-color-theme', colorTheme);
+    let theme = kpxc.settings['colorTheme'];
+    if (theme === 'system') {
```

Comment:
This same if sentence repeats in many files. It could be changed to a function inside `ui.js`, for example:
```
setTheme = function(theme, elem) {
    if (theme === 'system') {
        theme = window.matchMedia('(prefers-color-scheme: dark)').matches ? 'dark' : 'light';
    }
    elem.setAttribute('data-bs-theme', theme);
}
```

Then calling it like `setTheme(theme, document.documentElement);` etc.

Reply:
Hmm.. It seems like `ui.js` is only usable for content script functions? I tried it but I didn't find that it helped so much since I couldn't use it elsewhere.

I guess it is probably possible to include `ui.js` everywhere, but I haven't looked into it that much. I can try it later.

Reply:
How about creating a separate `theme.js`?